README.md

```diff
@@ -20,10 +20,85 @@ Living outside of the core product, Kyma allows you to be completely language-ag
+  ├── components                  # Source code of the application                                                   
+  ├── docs                        # Documentation
+  ├── installation                # Installation scripts     
+  ├── resources                   # Helm charts and Kubernetes resources for the Kyma installation
+  ├── tests                       # Acceptance tests
+  ├── tools                       # Source code of utilities used in the installation and testing    
+  ├── CODEOWNERS                  # Repository owners  
+  ├── CODE_OF_CONDUCT.md          # Rules of behavior for maintainers and contributors          
+  ├── CONTRIBUTING.md             # Rules of contribution
+  ├── Jenkinsfile                 # Integration job
+  ├── LICENSE                     # The document with terms for use, reproduction, modification, and distribution of Kyma code
+  ├── NOTICE.md                   # Copyright information
+  ├── README.md                   # The document with the project overview, structure, installation, and development steps
+  ├── azure.Jenkinsfile           # The file that builds and tests the Azure cluster
+  ├── ci.Dockerfile               # The file that verifies and tests the Docker image built
+  ├── governance.Jenkinsfile      # The file that validates external links in the documentation
+  └── orchestrator.Jenkinsfile    # The file that triggers Kyma integration and runs the CI files of all subprojects based on the introduced changes     
+  ```
+
 ## Installation
 
 Install Kyma [locally](docs/kyma/docs/031-gs-local-installation.md) and on a [cluster](docs/kyma/docs/032-gs-cluster-installation.md).
 
 ## Usage
 
 Kyma comes with the ready-to-use code snippets that you can use to test the extensions and the core functionality. See the list of existing examples in the [`examples`](https://github.com/kyma-project/examples) repository.
+
+## Development
+
+Develop on your remote repository forked from the original repository in Go.
+See the example that uses the [`ui-api-layer`](components/ui-api-layer) project located in the `components` directory in the `kyma` repository but applies to any Go project. This set of instructions uses the recommended [`git workflow`](https://github.com/kyma-project/community/blob/master/git-workflow.md).
+
+1. Fork the repository in GitHub.
+2. Clone the fork to your `$GOPATH` workspace. Use this command to create the folder structure and clone the repository under the correct location:
+
+```
+git clone git@github.com:{github-username}/kyma.git $GOPATH/src/github.com/{github-username}/kyma
+```
+3. Add a Git remote for the original repository:
+
+```
+git remote add upstream https://github.com/kyma-project/community.git
+```
+
+4. Run the `git fetch --all` command to fetch all remote branches.
+
+5. Set up the local `master` branch to track the remote master branch from the upstream repository:
+
+```
+git branch -u upstream/master master
+```
+
+6. Install dependencies.
```

Comment:
I'd convert the numbered list to separate sections (subheaders). Because the only thing that should be done in order, is installing dependencies before doing anything else.

Reply:
Aren't subheaders to much for such short instructions/steps that we want to keep simple? We can make bullet points out of the list if you still consider that essential?.